### DISCUSSION TITLE
Error while running SEBE (umep processor) - Urgent

### DISCUSSION THREAD

**Discussion by Frappetro** [UMEP-dev/UMEP#434] [Q&A]
Hi everyone!! I'm finally finishing my master degree thesis but I'm having trouble with running the SEBE simulation, in particular while running it, this python error comes out:

TypeError: can only concatenate str (not "NoneType") to str 
Traceback (most recent call last):
  File "C:\Users/Petro/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\UMEP\SEBE\sebe.py", line 472, in start_progress
    WriteMetaDataSEBE.writeRunInfo(self.folderPath[0], filepath_dsm, self.gdal_dsm, self.usevegdem,
  File "C:\Users/Petro/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\UMEP\SEBE\WriteMetaDataSEBE.py", line 27, in writeRunInfo
    file.write('Projected referece system: ' + srs.GetAttrValue('projcs'))
TypeError: can only concatenate str (not "NoneType") to str

I understand the problem could be related to the reference system but don't know ho to solve it!!
While running the Swedish tutorial i got no problem, but now seems the programm can't upgrade the text file with the correct info, it only acquires information about surface data and model domain, not utc, latitude or longitude...

I need to solve this problem and send the thesis, please help me!! 
Thank you in advance!

**Comment by biglimp** [comment]
Try change to a CRS on your DSM that are more common e.g. UTM. For e.g. west coast Sweden that would be EPSG:32632.

**Reply by Frappetro** [reply]
I'm currently working on a north Italy region, and I'm using ED50 UTM feseau 30 (IGNF:ED50UTM30) and ESPG: 23032 but still the same error.. Might be also the fact that i merged 4 different DSM togheter and during this process some informations have been lost? 
Thanks in advance for the fast reply!

**Reply by biglimp** [reply]
Can you attach your DSM here so that I can have a look.

**Reply by biglimp** [reply]
Your .asc file format (ESRI ASCII grid) does not include any CRS so you need to export it to something else. I recommend a geotiff (.tif). When you export you can set the CRS. I did this and checked if it aligned with OSM-data, and it did. 

Also, remember that you need local meteorological data for SEBE to produce reasonable results. If you dont have any, try to use the **Download data (ERA5)** - plugin in UMEP.

![image](https://user-images.githubusercontent.com/11908356/177321255-f09a4c7e-a8e5-472e-a780-7ff96f51f4a9.png)

**Reply by Frappetro** [reply]
Thank you so much! It worked, I just need to export it in another format as you said.
I also manage to create my local meteorological data from an .epw file.
You’re a lifesaver!!
